Cosmostation/Model/SupportPool.swift

```diff
@@ -9,11 +9,13 @@
 import Foundation
 
 public struct SupportPool {
+    var type: String!
```

Comment:
if you don't plan to mutate type once initialized you should consider to use let
`let`: constants not mutable
`var`: mutable properties